lib/tasks/trends.rake

```diff
@@ -29,4 +29,102 @@ namespace :trends do
     end
   end
 
+  desc 'Reduces amount of points for each trend group'
+  task :reduce => :environment do
+    start = Time.now
+
+    trends = Trend.pluck(:id)
+    trends_count = trends.length
+    current_record = 0
+
+    trends.each do |trend_id|
+      puts "Working on trend_id #{trend_id}, #{(current_record += 1)} of #{trends_count}"
+
+      current_interval = TrendCounter.where(trend_id: trend_id).order(:created_at).first
```

Comment:
Pretty much - it won't change a thing, unless there are intervals that can be consolidated. There shouldn't be in the common use case, but if there will be - they would be consolidated.